src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1918,6 +1918,13 @@ private function registerMailerConfiguration(array $config, ContainerBuilder $co
 
         $loader->load('mailer.xml');
         $container->getDefinition('mailer.default_transport')->setArgument(0, $config['dsn']);
+
+        $recipients = $config['envelope']['recipients'] ?? null;
+        $sender = $config['envelope']['sender'] ?? null;
```

Comment:
maybe you can move this to the configuration using `->defaulltNull()` ?

Reply:
For the `recipients` this will not work since: 

> InvalidArgumentException : framework.mailer.envelope.recipients: the default value of an array node has to be an array.